xpk/xpk.py

```diff
@@ -1450,6 +1459,18 @@ def workload_name_type(value, pat=re.compile(r'[a-z]([-a-z0-9]*[a-z0-9])?')):
     help='The number of slices to run the job on, defaults to 1.',
     required=True,
 )
+cluster_create_optional_arguments.add_argument(
+  '--cluster-type',
```

Comment:
--cluster-type also needs to be an argument for 'xpk cluster list'. Or actually maybe cluster list should show both zonal and regional clusters. That would be nicer?

I don't think there is a single argument for this, so the ask would be to run two commands one with --zone and one with --region in `xpk cluster list'

Reply:
Also to reiterate, `cluster delete` will need the argument as well